README.md

```diff
@@ -1 +1,85 @@
-# budget-app
\ No newline at end of file
+# Budget App
+
+The Budget App is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+![imagen](./home.jpeg)
+
+## Built With
+
+- ![Ruby](https://img.shields.io/badge/Ruby-20232A?style=for-the-badge&logo=ruby&logoColor=61DAFB)
+
+## Live Demo
+
+[Demo Available Here](https://sheltered-escarpment-74586.herokuapp.com/)
+
+## Getting Started
+
+### Installation
+
+1. Clone the repo https://github.com/Richie9000/budget-app.git
+
+### Usage
+
+1. Install ruby gems dependencies
+
+   ```sh
+   bundle install
+   ```
+
+2. Install node dependencies
+
+   ```sh
+   npm install
+   ```
+
+3. Database creation
+
+   ```
+   rails db:setup
+   ```
+
+4. Run the web server
+
+   ```sh
+   rails server
+   ```
+
+5. Access to the website in a browser with the following link
+
+   ```sh
+   localhost:3000
+   ```
+
+### Test
+
+1. For testing run:
+
+   ```sh
+   rspec
+   ```
+
+## Acknowledgements
```

Comment:
- [ ] It's a nice and smart thing to give appropriate credit to **[Gregoire Vella on Behance](https://www.behance.net/gregoirevella)**, the author of [the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Kindly ensure to add this under the **Acknowledgements** section of your ReadMe.

Reply:
:heavy_check_mark: This is solved.